File: app/src/main/java/foocafe/org/foocafe/entities/RegistrationCredential.java
package foocafe.org.foocafe.entities;


import com.google.gson.annotations.SerializedName;

public class RegistrationCredential {

    @SerializedName("email")
    String email;

    @SerializedName("password")
    String password;

    public RegistrationCredential(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }
}

File: app/src/main/java/foocafe/org/foocafe/entities/RegistrationCredential.kt
package foocafe.org.foocafe.entities


import com.google.gson.annotations.SerializedName

class RegistrationCredential(email: String, password: String) {

    @SerializedName("email")
    var email: String
        internal set

    @SerializedName("password")
    var password: String
        internal set

    init {
        this.email = email
        this.password = password
    }
}